### PR TITLE
Add release drafter CI

### PR DESCRIPTION
**Pull request by neilzar**
**When merged this pull request will:**
- _Title_

### CODE REVIEW

**Comment by CreepPork** [comment]
Could you add an example generated changelog from the drafter (e.g. v1.7.0)?

I'm worried that it might be too unclean to be of any use where you'd have to go and still edit every single line where it might be quicker to do it by hand altogether.

**Comment by jonpas** [comment]
Example: https://github.com/acemod/ACEX/releases/tag/v3.5.1 (the only edited thing is required CBA version)

Release Drafter still has some bugs, it will duplicate entries for PRs with multiple tags. And it also works based on date alone, so hotfixes will erase it (easy to overcome if you just copy it somewhere else). It will also always use only last draft, no matter who made it.

And finally, it was very useful for latest ACE3 and ACEX releases, you just need a standard for PR titles to keep it clean.

**Comment by neilzar** [comment]
> Release Drafter still has some bugs, it will duplicate entries for PRs with multiple tags. 
This was apparently already fixed in the latest release, 26 days ago.

I can't find an issue on the release-drafter repo about the erasing issue.

**Comment by jonpas** [comment]
Because it's not an issue, it's how it works. It will always draft from latest release to current.

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
 
 change-template: '- $TITLE ($NUMBER)'
 template: |
-  _Requires CBA version X.Y.Z or later and Arma 3 version X.Y or later._
+  _Requires CBA Version X.Y.Z or later and Arma 3 Version X.Y or later._
+
+  DESCRIPTION HERE
 
-  ## Changelog Summary
+  ## Change Log Summary
```
Comment (neilzar) [inline .github/release-drafter.yml:29]: Changelog is actually the spelling used on wiki and in many other places I encounter. I don't see Change Log being used as much.

Comment (mharis001) [reply .github/release-drafter.yml:29]: It has been used in all releases so far.

**Pull request merged**